package.json

```diff
@@ -33,10 +33,11 @@
     "build:svgs": "lerna run svgs --scope bpk-svgs",
     "build": "npm run build:tokens && npm run build:svgs && lerna run build && (cd ./native && npm run link)",
     "storybook:dist": "build-storybook -c .storybook -o dist/storybook",
+    "ios-api:dist": "(cd backpack-ios && bundle install && npm i && gulp && rake \"docs['../dist/ios/']\")",
```

Comment:
Running `npm i` and `gulp` here is redundant as we commit the whole output of gulp. We could save time by skipping this step